src/useMutationObserver.ts

```diff
@@ -0,0 +1,69 @@
+import useEffect from './useIsomorphicEffect'
```

Comment:
```suggestion
import useIsomorphicEffect from './useIsomorphicEffect'
```

maybe? otherwise it's less clear what kind of thing we're doing